src/Symfony/Component/Validator/Command/DebugCommand.php

```diff
@@ -0,0 +1,195 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Dumper;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Mapping\ClassMetadataInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * A console command to debug Validators information.
+ *
+ * @author Loïc Frémont <lc.fremont@gmail.com>
+ */
+class DebugCommand extends Command
+{
+    protected static $defaultName = 'debug:validator';
+
+    private $validator;
+
+    public function __construct(ValidatorInterface $validator)
+    {
+        parent::__construct();
+
+        $this->validator = $validator;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->addArgument('class', InputArgument::REQUIRED, 'A fully qualified class name or a path')
+            ->addOption('show-all', null, InputOption::VALUE_NONE, 'Show all classes event if they have no validation constraints')
+            ->setDescription('Displays validation constraints for classes')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command dumps the validation constraints configuration for all classes in
+`src/Entity` directory.
```

Comment:
As the argument is required, there is no special case for src/Entity, that should be removed from the description.